Assets/MixedRealityToolkit.Services/InputSimulation/Editor/MixedRealityToolkit.Services.InputSimulation.Editor.asmdef

```diff
@@ -4,7 +4,8 @@
         "Microsoft.MixedReality.Toolkit",
         "Microsoft.MixedReality.Toolkit.Core.Inspectors",
         "Microsoft.MixedReality.Toolkit.Core.Utilities.Editor",
-        "Microsoft.MixedReality.Toolkit.Services.InputSimulation"
+        "Microsoft.MixedReality.Toolkit.Services.InputSimulation",
+        "Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality"
```

Comment:
there should not be dependencies between services and platforms. was there a reason the WMR provider needed to be added?

Reply:
resolved itself